verilog/formatting/formatter_test.cc

```diff
@@ -5633,7 +5633,7 @@ static constexpr FormatterTestCase kFormatterTestCases[] = {
     {"module mp ;property p1 ; a|->## [0  : 1]  b;endproperty endmodule",
      "module mp;\n"
      "  property p1;\n"
-     "    a |-> ##[0 : 1] b;\n"  // limit to one space, symmetrize
+     "    a |-> ##[0:1] b;\n"  // limit to one space, symmetrize
```

Comment:
I think this is ok, even though semantically, it is a delay range context, not a declared range or indexing.

Also update // comment

Reply:
Done